### PR TITLE
Add redux in react components

### PR DESCRIPTION
**Pull request by Victoria129**
- I have added redux to react components.
- I have also reorganized my file structure.

### CODE REVIEW

File: src/components/BookForm/BookForm.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    setTitle('');
+  };
+
+  return (
+    <form onSubmit={onSubmit}>
+      <input value={title} onChange={(e) => setTitle(e.target.value)} type="text" placeholder="Book Title" />
+      <input value={author} onChange={(e) => setAuthor(e.target.value)} type="text" placeholder="Author" />
+      <button type="submit">Add Book</button>
+    </form>
+  );
```
Comment (Ugboaja-Uchechi) [inline src/components/BookForm/BookForm.js:28]: - [ ] Great job with your project 👍.  The form for adding books isn't present on the webpage, making it impossible to add books. Kindly fix this error 🙏

**Review by Ugboaja-Uchechi — CHANGES_REQUESTED** [review]
Hi @Victoria129, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- Linters are passing ✔️ 
- README is descriptive 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/BookForm/BookForm.js (hunk 1 above)
Comment (Abenezer-Tilahun) [reply src/components/BookForm/BookForm.js:28]: Fixed.

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @Victoria129 
 
Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Git-Flow.
✔️ Professional README  
✔️ Good Commit Messages
✔️ Good job for Implemented Suggested changes . 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**